draft-ietf-dprive-dnsoquic.md

```diff
@@ -470,8 +474,11 @@ assume that the maximum message size is 65535 bytes.
 
 For the stub to recursive resolver scenario, the authentication requirements
 are the same as described in DoT {{?RFC7858}} and "Usage Profiles for DNS over
-TLS and DNS over DTLS" {{!RFC8310}}. There is no need to authenticate the
-client's identity in either scenario.
+TLS and DNS over DTLS" {{!RFC8310}}. {{?RFC8932}} states that DNS privacy
+services SHOULD provide credentials that clients can use to authenticate the
+server. Given this, and to align with the authentication model for DoH, DoQ stubs 
+SHOULD use a Strict authentication profile. There is no need
+to authenticate the client's identity.
 
```

Comment:
May need to rephrase "there is no need to authenticate the client", especially in the recursive to authoritative scenario.